### PR TITLE
TUNIC: Error catching for logic bugs in ER

### PR DESCRIPTION
**Pull request by ScipioWright**
## What is this fixing or adding?
Currently, if there was an error encountered where a location is unreachable in entrance rando, it would raise an exception that isn't the most clear to a host.
Now, it'll log a warning and make the entrance hint "Inaccessible". This is so that gen can continue, since I wouldn't want something like this that doesn't always lead to a game that can't be beaten to stop games from happening.

There aren't any currently known bugs in the entrance rando logic, this is mostly a just in case.

## How was this tested?
Test gens.
Easy way to test is to go to the line in er_scripts that is `while len(connected_regions) < len(non_dead_end_regions):` and change it to `while len(connected_regions) < len(non_dead_end_regions) - 10:`, with the entrance rando option enabled in the yaml. It might take a few tries for it to actually proc.

## If this makes graphical changes, please attach screenshots.
N/A

### CODE REVIEW

File: worlds/tunic/__init__.py
Comment (NewSoupVi) [inline worlds/tunic/__init__.py:255]: This is not nearly funny enough, needs to have at least 3 layers of self-deprecation or flirty vibes /j

**Review by NewSoupVi — APPROVED** [review]
Seems completely risk-free

File: worlds/tunic/__init__.py
Comment (ScipioWright) [reply worlds/tunic/__init__.py:255]: So true

**Review by Berserker66 — APPROVED** [review]

**Comment by ScipioWright** [comment]
For worlds with multiple maintainers, are we meant to get approvals from all of the maintainers? I'm the main one doing all of the entrance rando stuff, so this doesn't really have anything to do with Silent

**Pull request merged**
